.env.example

```
@@ -0,0 +1,14 @@
+# Rename this file to `.env` and fill in values for production or development.
+# Expo automatically exposes EXPO_PUBLIC_* at build time.
+
+# Toggle local emulators in development. For production builds set this to false or remove it.
+EXPO_PUBLIC_USE_FIREBASE_EMULATORS=false
+
+# Firebase web config
+EXPO_PUBLIC_FIREBASE_API_KEY=
+EXPO_PUBLIC_FIREBASE_AUTH_DOMAIN=
+EXPO_PUBLIC_FIREBASE_PROJECT_ID=locker-room-talk-app
+EXPO_PUBLIC_FIREBASE_STORAGE_BUCKET=
+EXPO_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_FIREBASE_MESSAGING_SENDER_ID key should go before the EXPO_PUBLIC_FIREBASE_PROJECT_ID key

(UnorderedKey)